### ISSUE TITLE
Cannot package web application to executable fat jar!

### ISSUE THREAD

**Issue by yujinping** [oblac/joy-boot#2]
@igr 
I use joy-boot for local machine to test some web application.

In fact, what I need is the ability to package the jodd web application(including the dependent jars) into a fat jar file, and then run as the way like springboot(just using cmd  ```java -jar test.jar``` to run the web app of jodd), do you have any good ideas?

**Comment by igr** [comment]
Yes - just use some fatjar plugin - it should be enough. Jodd-boot has all dependencies in the gradle, as usually, so just pack it with such plugin. This should work.
